kconfig_no_jffs2.conf

```diff
@@ -0,0 +1 @@
+# CONFIG_JFFS2_FS is not set
```

Comment:
Yes it does. For the record... here is how it works.

[merge_config.sh](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/scripts/kconfig/merge_config.sh) reads all the supplied files, and considers the lines that match "CONFIG_something =" and "# CONFIG_something" (with exactly *one* space between the hash sign and the config name!). It merges all these values, reporting conflicts as it goes, into a unique file.

Then, this file is processed by "make alldefconfig" which will copy the supplied definitions into the final `.config` file as long as dependencies are not violated. In any case, `merge_config.sh` will report any discrepancy between what was requested and what is actually used. Variables that are unspecified take their default values (alldefconfig) but it is also possible to ask for them to be disabled (allnoconfig, `merge_config.sh -n`).

Let's compare the final .config with or without this `kconfig_no_jffs2.conf`:
```
# Use only the "preempt" fragment
$ rm -f ../linux/.config ; make linux-defconfig KCONFIGS=kconfig_preempt.conf >/dev/null
$ grep CONFIG_JFFS2_FS ../linux/.config
CONFIG_JFFS2_FS=y
CONFIG_JFFS2_FS_DEBUG=0
CONFIG_JFFS2_FS_WRITEBUFFER=y
# CONFIG_JFFS2_FS_WBUF_VERIFY is not set
# CONFIG_JFFS2_FS_XATTR is not set
$

# Use both the "preempt" and "no_jffs2" fragments
$ rm -f ../linux/.config ; make linux-defconfig >/dev/null
$ grep CONFIG_JFFS2_FS ../linux/.config
# CONFIG_JFFS2_FS is not set
$ 
```